src/g42-main.cpp

```diff
@@ -304,6 +305,15 @@ int main(int argc, const char **argv) { // the main() function
 	vector<string> argt; // args (without exec name)
 	for (int i=1; i<argc; ++i) argt.push_back(argv[i]);
 	bool early_debug = contains_value(argt, "--d");
+	std::setlocale(LC_ALL, "en_US.UTF-8");
```

Comment:
needs to be in try-catch   setlocale can throw ( e.g. https://gcc.gnu.org/bugzilla/show_bug.cgi?id=70493 )

std::cerr<<"Error: setlocale."<<std::endl;

Reply:
seems done (as of 7fa89ee)